### ISSUE TITLE
Saved searches wont show up

### ISSUE THREAD

**Issue by 4scet** [ghostscript3r/poe-trade-official-site-enhancer#7]
Hey, as in #3 said, saved searches are "disappearing" using official /trade site. However after exporting I can see my custom saved searches in .txt file.
The problem seem to be up for few months, any updates?

**Comment by Azdingo** [comment]
What worked for me is opening that window 2 times... The first time it's all blank. Close it and reopen it.

**Comment by tonpix** [comment]
Same - if I see the list of saved searches blinking once, or straight empty, I just close it and open again. It happens on Chrome, on Firefox I don't seem to have issue with blinking/empty lists. So I'm guessing it's sort of issue with Chrome or one of its' extensions I'm using. (disabling adblocker doesn't help)

**Comment by ghostscript3r** [comment]
Haven't been able to reproduce this yet (I only use chrome generally and it never happened to me). Could one of you try to activate the debugger? You have to go to TamperMonkey Dashboard, click on the script and edit it setting the DEBUG variable to true (https://imgur.com/ekHj1Yv). 
After that refresh the page, hit F12 to open the js console. Click the clear console button (see the arrow here https://imgur.com/bjbYGXX) so there's less noise, and try to open the load window. You should see a lot of debug text. Please copy or export it after an empty window so I can try to pinpoint the problem

**Comment by tonpix** [comment]
I've selected console content and copied as text, doesn't seem to keep formatting, let me know if there's a better way to save console output. https://pastebin.com/Pji4RHC0

+ screenshot if it helps somewhat
![image](https://user-images.githubusercontent.com/13473971/74484763-87a6e700-4eb9-11ea-8c97-8c7a8f6d04eb.png)

**Comment by ghostscript3r** [comment]
Very strange behavior. It seems it finds the list but it's empty. And if you close and reopen it shows the saves right?

**Comment by tonpix** [comment]
> Very strange behavior. It seems it finds the list but it's empty. And if you close and reopen it shows the saves right?

Yes. Then list keeps opening properly, until the trade page is reloaded in browser or new tab is opened

**Comment by Asymat** [comment]
I have the same problem. Multi-live search also have the same problem.

Both doesn't show saved searches the first time you open the saved/multi window. You have to close and reopen saved/multi window to finally see your previous saved searches.

**Comment by tonpix** [comment]
I can confirm it again as well in Delirium league. I saw new features, but seems this issue haven't got squashed yet, unfortunately. It's bit distracting when you have to update filters quite often.

How's the bug status?

**Comment by xeske** [comment]
Possible workaround: "Save search"->"Close"->"Load search". List returns.

**Comment by ghostscript3r** [comment]
Please try new release 1.3.420. Not sure the problem it's solved because I can't reproduce. I added a retry if the list is empty but saved searches have been found. I added some specific debug logs anyway, so it should be easier to narrow it down now.

**Comment by tonpix** [comment]
> Please try new release 1.3.420. Not sure the problem it's solved because I can't reproduce. I added a retry if the list is empty but saved searches have been found. I added some specific debug logs anyway, so it should be easier to narrow it down now.

Unfortuntately the problem persists. 

Any ideas how can I help you? There's the new console output with debug enabled: https://pastebin.com/xLmwdZFG (let me know if it was copied properly)

**Comment by ghostscript3r** [comment]
What you did was perfect, helped me narrow it down more. It seems there's a problem while getting data from localStorage. Can you do it again with 1.3.422? Added few more debug lines to try to pinpoint where it does not return anything. I fear it's a chrome issue and probably very time consuming to solve, so I need to be sure before trying to find a solution.

**Comment by tonpix** [comment]
Sure, updated with new pastebin https://pastebin.com/q9BjYezL

**Comment by tonpix** [comment]
Hey, any progress?

**Comment by Asymat** [comment]
https://amp.reddit.com/r/pathofexile/comments/fhfkng/a_better_trading_experience_chrome_extension/
Tried it. Loved it

**Comment by ghostscript3r** [comment]
closing, check out new version of the script, this problem should be fixed as well
